src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java

```diff
@@ -661,6 +672,62 @@ public void configureHotspot(JSONArray args, final CallbackContext callback) thr
         return response;
     }
 
+    /**
+     * Returns a list of the configured networks on the device
+     */
+    private void getConfiguredNetworks(CallbackContext callback) throws JSONException {
+        Log.i(LOG_TAG, "Running getConfiguredNetworks() ");
+        WifiManager wifiManager = (WifiManager) this.cordova.getActivity()
+                .getApplication()
+                .getApplicationContext()
+                .getSystemService(Context.WIFI_SERVICE);
+        List<WifiConfiguration> networks = wifiManager.getConfiguredNetworks();
+        JSONArray networksArray = new JSONArray();
+
+        if (networks != null) {
+            for (WifiConfiguration wifiConfig : networks) {
+                networksArray.put(toJSON(wifiConfig));
+            }
+        }
+
+        callback.success(networksArray);
+    }
+
+    /**
+     * Parse the WifiConfiguration object to JSON,
+     * it's parsing only the necessary fields, the full list is in:
+     * https://developer.android.com/reference/android/net/wifi/WifiManager.html#getConfiguredNetworks()
+     * https://developer.android.com/reference/android/net/wifi/WifiConfiguration.html
+     * 
+     */
+    private static JSONObject toJSON(WifiConfiguration wifiConfig) throws JSONException {
```

Comment:
The name seems to suggest that you'll be returning a JSON representation of the `HotSpotPlugin`. I think a more descriptive name like `configToJson` might be more appropriate.

Reply:
Good point, I've updated the function name